staging/src/k8s.io/apiserver/pkg/endpoints/installer_test.go

```diff
@@ -302,6 +302,8 @@ func TestConvertAPIResourceToDiscovery(t *testing.T) {
 				{
 					Resource: "cronjobs",
 					Scope:    apidiscoveryv2beta1.ScopeNamespace,
+					// aggregator populates this to avoid nil panics
```

Comment:
nit: Not just the aggregator, [the standard apiserver](https://github.com/kubernetes/kubernetes/blob/cb56c4c627fc627b2eb1ba69a435cc3f205f7363/staging/src/k8s.io/apiserver/pkg/endpoints/groupversion.go#L122) also attempts to populate it?